hcloud/load_balancer.go

```diff
@@ -148,6 +169,18 @@ type LoadBalancerTargetServer struct {
 	Server *Server
 }
 
+// LoadBalancerTargetLabelSelector configures a Load Balancer target
+// pointing to many servers.
```

Comment:
I think the comment should be more clear in that a LoadBalancerTargetLabelSelector does not specify multiple servers directly, but rather a label selector query which _may_ resolve to many servers.